### PR TITLE
Dynamic sizing nodes

### PR DESCRIPTION
**Pull request by tomislater**
It is part of #6 issue. There is still a room for improvement, for instance we could put more clusters in a row.

screenshot 2019-03-03 at 22 41 00
screenshot 2019-03-03 at 22 39 56

### CODE REVIEW

File: app/src/node.js [hunk 1]
```diff
@@ -103,33 +105,37 @@ export default class Node extends PIXI.Graphics {
...
-                px += 13
-                if (px > 90) {
-                    px = 24
-                    py += 13
-                }
-            }
-        }
-        px = 24
-        py = 100
-        for (const pod of pods) {
```
Comment (tomislater) [inline app/src/node.js]: We do not need this loop anymore.

File: kube_ops_view/mock.py [hunk 2]
```diff
@@ -106,13 +108,18 @@ def query_mock_cluster(cluster):
             for c in p["containers"]:
                 usage_cpu += int(c["resources"]["usage"]["cpu"].split("m")[0])
                 usage_memory += int(c["resources"]["usage"]["memory"].split("Mi")[0])
+
```
Comment (tomislater) [inline kube_ops_view/mock.py]: I can add it in another PR if you want. If you want to keep it of course :)

**Comment by tomislater** [comment]
Hmm, https://app.codacy.com/app/henning/kube-ops-view/pullRequest?prid=3204677 what should I do regarding this?

**Comment by hjacobs** [comment]
Thanks for your contribution! I won't be able to look at it today I think, but will check later.

**Comment by hjacobs** [comment]
The PR looks good, but we still have the potential overlapping of system vs user pods:
screenshot_2019-03-06 kubernetes operational view 2017 0 dev1

**Comment by hjacobs** [comment]
Looks great :smile: 

Here two real clusters side-by-side (one empty cluster and one non-empty with more pods) 
screenshot_2019-03-06_20-25-32

**Comment by tomislater** [comment]
Hmm, we can also scale masters separately. If you have a very large number of pods in one node then masters might look strange. I mean, we can waste a lot of space then. I can do it in the near future.

**Comment by hjacobs** [comment]
Released: https://github.com/hjacobs/kube-ops-view/releases/tag/0.10

**Pull request merged**